airflow/www/templates/airflow/dag.html

```diff
@@ -310,6 +310,10 @@ <h4>Task Actions</h4>
                   <input type="checkbox" value="true" name="success_downstream" autocomplete="off">
                   Downstream
                 </label>
+                <label class="btn btn-default btn-sm active" title="Automatically clear blocked downstream tasks">
```

Comment:
```suggestion
                <label class="btn btn-default btn-sm active" title="Automatically clear downstream tasks that were in 'upstream_failed' state">
```

(If that's actually what it does?

Reply:
This change is no longer needed.